agent_deploy/kubernetes/sysdig-agent-daemonset-v1.yaml

```diff
@@ -3,13 +3,17 @@
 # We recommend to use v2 daemonset which instead leverages Kubernetes
 # best practices like Secrets and ConfigMaps
 ##########################################################################
-apiVersion: extensions/v1beta1
+# apiVersion: extensions/v1beta1  # If you are in Kubernetes version 1.8 or less please use this line instead of the following one
+apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: sysdig-agent
   labels:
     app: sysdig-agent
 spec:
+  selector:
```

Comment:
this change seems unrelated to what the PR states. and btw, why is this required? it hasn't been so far afaik.

Reply:
Because the new api require to have a selector for all objects using it. You can not change the api and that's all, at least for deployments and daemonsets

Reply:
Error you get if you try to change only the api and don't add a selector: 

# * : Invalid value: "The edited file failed validation": ValidationError(DaemonSet.spec): missing required field "selector" in io.k8s.api.apps.v1.DaemonSetSpec